com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -210,144 +283,67 @@ void GetAndSetData(int i)
 
             var conv = m_CoreConvertersList[i];
 
+            m_ConverterStates[i].isLoading = true;
+
             // This should also go to the init method
             // This will fill out the converter item infos list
-            conv.OnInitialize(initCtx);
-
-            // Set the item infos list to to the right index
-            m_ItemsToConvert[i] = converterItemInfos;
-            m_ConverterStates[i].items = new List<ConverterItemState>(converterItemInfos.Count);
-
-            // Default all the entries to true
-            for (var j = 0; j < converterItemInfos.Count; j++)
-            {
-                string message = string.Empty;
-                Status status;
-                bool active = true;
-                // If this data hasn't been filled in from the init phase then we can assume that there are no issues / warnings
-                if (string.IsNullOrEmpty(converterItemInfos[j].warningMessage))
-                {
-                    status = Status.Pending;
-                }
-                else
+            conv.OnInitialize(initCtx,
+                () =>
                 {
-                    status = Status.Warning;
-                    message = converterItemInfos[j].warningMessage;
-                    active = false;
-                    m_ConverterStates[i].warnings++;
-                }
+                    // Set the item infos list to to the right index
+                    m_ItemsToConvert[i] = converterItemInfos;
+                    m_ConverterStates[i].items = new List<ConverterItemState>(converterItemInfos.Count);
 
-                m_ConverterStates[i].items.Add(new ConverterItemState
-                {
-                    isActive = active,
-                    message = message,
-                    status = status,
-                    hasConverted = false,
-                });
-            }
+                    // Default all the entries to true
+                    for (var j = 0; j < converterItemInfos.Count; j++)
+                    {
+                        string message = string.Empty;
+                        Status status;
+                        bool active = true;
+                        // If this data hasn't been filled in from the init phase then we can assume that there are no issues / warnings
+                        if (string.IsNullOrEmpty(converterItemInfos[j].warningMessage))
```

Comment:
I dont fully understand the logic here. Having no warning message means the item is still pending? 
We can look into this for a future PR.